_sass/blocks/_landing.scss

```diff
@@ -88,43 +50,71 @@
     }
   }
 
-  &.oil-gas,
-  &.coal,
-  &.minerals,
-  &.renewables {
-    background-size: 100%;
-    height: 310px;
-
-    .landing-description {
-      @include span-columns(7);
-    }
-  }
-
   @include respond-to(medium-up) {
-    &.oil-gas {
+    &.overview {
+      @include span-columns(6);
       @include shift(2);
     }
+  }
 
-    &.coal {
-      @include omega();
-    }
+  .landing-heading {
+    display: block;
+  }
+}
 
-    &.minerals {
-      @include shift(2);
-    }
+.landing-oil_gas {
+  background: url('../img/how-it-works/cards/oil-gas.png') bottom left no-repeat;
+}
 
-    &.renewables {
-      @include omega();
-    }
+.landing-coal {
+  background: url('../img/how-it-works/cards/coal.png') bottom left no-repeat;
+}
 
-    &.overview {
-      @include span-columns(5);
-      @include shift(2);
-    }
+.landing-minerals {
+  background: url('../img/how-it-works/cards/minerals.png') bottom left no-repeat;
+}
+
+.landing-renewables {
+  background: url('../img/how-it-works/cards/renewables.png') bottom left no-repeat;
+}
+
+.landing-oil_gas,
+.landing-coal,
+.landing-minerals,
+.landing-renewables {
+  background-size: 100%;
+  height: 230px;
+
+  p {
+    width: 63%;
+    padding-top: $base-padding;
   }
 
+  @include respond-to(medium-up) {
```

Comment:
Expected item on line 93 to appear before line 88. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets